connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntityStatusTranslator.java

```diff
@@ -88,6 +90,22 @@ public void translate(ServerEntityStatusPacket packet, GeyserSession session) {
                 break;
             case LIVING_DEATH:
                 entityEventPacket.setType(EntityEventType.DEATH);
+                if (entity.getEntityType() == EntityType.THROWN_EGG) {
+                    for (int i = 0; i < 6; i++) {
+                        LevelEventPacket particlePacket = new LevelEventPacket();
+                        particlePacket.setType(LevelEventType.PARTICLE_GENERIC_SPAWN);
+                        particlePacket.setData(22544397);
```

Comment:
Is there any sort of constant where this value is stored? Not sure how I feel with this magic number just sitting here.